src/App.tsx

```diff
@@ -1,25 +1,43 @@
-import React from "react";
-import logo from "./logo.svg";
+import React, { useEffect, useState } from "react";
 import "./App.css";
 
 function App() {
+  const [items, setItems] = useState([]);
+  const [randomItem, setRandomItem] = useState();
+
+  useEffect(() => {
+    fetch("https://jsonplaceholder.typicode.com/todos")
+      .then((response) => response.json())
+      .then((json) => {
+        setItems(json);
+      });
+  }, []);
+
+  useEffect(() => {
+    fetch("https://jsonplaceholder.typicode.com/todos")
+      .then((response) => response.json())
+      .then((json) => {
+        JSON.stringify(
+          setRandomItem(json[Math.floor(Math.random() * json.length)])
+        );
+      });
+  }, [randomItem]);
```

Comment:
`randomItem` will never be equal new version of `randomItem` since it's an object.

Reply:
Done